packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -125,6 +129,10 @@ export const UnitRow = ({
     l10n.getString('row-defaults-action-disable', null, 'Disable');
   ctaText = ctaText || (headerValue ? localizedCtaChange : localizedCtaAdd);
 
+  if (disabled) {
+    ctaText = l10n.getString('row-defaults-action-disable', null, 'Disable');
```

Comment:
Hmmm, I don't think we want the button itself to read "Disable" or "Disabled", Vesta just mentioned to me that she wanted the buttons (well, links, but I know a link can't be "disabled") "greyed out." I think we want the button to be disabled, but still read "Create" or "Add" so the user can see what the disabled action _is_. I think that's also better for screenreaders, because otherwise I believe (would need to double check on when "title" is read) this button will read "button, disabled, disabled, [title]" while reading "button, Create, disabled, [title]" is going to give those users slightly more context for what they've landed on.

Reply:
I can do this. I must say, however, I have some concerns about the mobile experience here. The grayed out button isn't super obvious and the title (ie tooltip) doesn't really do much for mobile users, since the hover event isn't really a thing.